### PR TITLE
keyworker results and profile components.  wip requires tests and fur…

### PR DESCRIPTION
**Pull request by lbennett-moj**
…ther work

### CODE REVIEW

File: src/KeyworkerProfile/container.js [hunk 1]
```diff
@@ -13,10 +14,15 @@ class KeyworkerProfileContainer extends Component {
     this.displayError = this.displayError.bind(this);
     this.getKeyworkerList = this.getKeyworkerList.bind(this);
     this.gotoKeyworkerResults = this.gotoKeyworkerResults.bind(this);
+    this.gotoKeyworkerProfile = this.gotoKeyworkerProfile.bind(this);
   }
 
   componentWillMount () {
-    this.props.setCurrentPageDispatch(1);
+    if (this.props.paramStaffId) {
+      this.gotoKeyworkerProfile(this.props.paramStaffId);
```
Comment (lbennett-moj) [inline src/KeyworkerProfile/container.js]: an approach for deep linking and linking from a list of results - can discuss

File: src/KeyworkerProfile/container.js [hunk 2]
```diff
@@ -45,6 +63,32 @@ class KeyworkerProfileContainer extends Component {
...
+    return response.data;
+  }
+
+  async getKeyworkerDetails (agencyId, staffId) {
+    const response = await axiosWrapper.get('/keyworker', {
+      headers: {
+        jwt: this.props.jwt
+      },
+      params: {
+        agencyId: agencyId,
```
Comment (steverendell) [inline src/KeyworkerProfile/container.js]: Atm we dont need the agency id

File: src/KeyworkerProfile/container.js (hunk 1 above)
Comment (steverendell) [reply src/KeyworkerProfile/container.js]: Looks good

File: src/KeyworkerProfile/container.js (hunk 2 above)
Comment (steverendell) [reply src/KeyworkerProfile/container.js]: Also this is the same api call as getting the KW name and no of allocations. But I tihink this is ok for now until we get the reqs of what KW details are actually needed

File: src/redux/actions/actionTypes.js [hunk 3]
```diff
@@ -15,3 +15,5 @@ export const SET_PRISONER_SEARCH_ALLOCATION_STATUS = 'SET_PRISONER_SEARCH_ALLOCA
 export const SET_PRISONER_SEARCH_HOUSING_LOCATION = 'SET_PRISONER_SEARCH_HOUSING_LOCATION';
 export const SET_KEY_WORKER_SEARCH_TEXT = 'SET_KEY_WORKER_SEARCH_TEXT';
 export const SET_KEY_WORKER_SEARCH_RESULTS_LIST = 'SET_KEY_WORKER_SEARCH_RESULTS_LIST';
+export const SET_KEY_WORKER_ALLOCATION_LIST = 'SET_KEY_WORKER_ALLOCATION_LIST';
+export const SET_KEY_WORKER = 'SET_KEY_WORKER';
```
Comment (steverendell) [inline src/redux/actions/actionTypes.js]: maybe SET_KEY_WORKER_PROFILE (or DETAILS?) is more descriptive?

**Pull request merged**